frontend/src/data/Breakdowns.ts

```diff
@@ -44,23 +44,29 @@ function demographicBreakdownToCol(
 
 export class Breakdowns {
   geography: GeographicBreakdown;
-  // Note: this assumes only one demographic breakdown at a time. If we want to
-  // support more later we can refactor this to multiple boolean fields.
-  demographic?: DemographicBreakdown;
   // We may want to extend this to an explicit type to support variants for
   // day/week/month/year.
   time?: boolean;
-
+  race?: boolean;
+  race_nonstandard?: boolean;
+  age?: boolean;
+  sex?: boolean;
```

Comment:
What do you think about making these booleans non-optional and then defaulting them to false in the constructor? Avoids the need to add `!!` everywhere they're used

Reply:
done- I added a new method on Breakdowns to keep the string in unsupported breakdown error the same - I think it's easier to read when we don't list which are set to false.